<launch>
    <group ns="encoder">
        <param name="tickPerMeter" value="97.94" />
        <param name="minVelocity" value="0.05" />
    </group>
    <param name="batteryVoltage" value="7.76"/>
    <node pkg="rmcore" name="experiment_tools" type="experiment_tools" required="true">
        <rosparam ns="leftEncoder">
            pin: 10
            collabrateData: [1.0221, 1.0068, 1.0060, 1.0062, 0.9823, 0.9754, 0.9759, 0.9698, 0.9516, 0.9687, 0.9721, 0.9796, 0.9844, 1.0142, 1.0238, 1.0386, 1.0346, 1.0398, 1.0336, 1.0323]
        </rosparam>
        <rosparam ns="rightEncoder">
            pin: 4
            collabrateData: [0.9551, 0.9640, 0.9745, 0.9727, 0.9865, 1.0023, 1.0068, 1.0071, 1.0011, 1.0418, 1.0416, 1.0438, 1.0456, 1.0355, 1.0238, 1.0005, 0.9924, 0.9892, 0.9755, 0.9588]
        </rosparam>
        <param name="leftMotor/device" value="/dev/gpiopwm/18"/>
        <param name="rightMotor/device" value="/dev/gpiopwm/13"/>
        <rosparam ns="imu">
            interruptPin: 14
            sampleRate: 500.0
            dlpfMode: 0
            gyroOffset:     [  3.65464,  -3.02076,  0.670199]
            gyroCorrectMat: [   0.995762,           0,   0.0919717,
                             0.000199762,    0.999998, -0.00216279,
                              -0.0919715,  0.00217199,    0.995759]


        </rosparam>
    </node>
</launch>
